test/mason/mason-help-scope/masonHelpScope.chpl

```diff
@@ -0,0 +1,21 @@
+
+use MasonUtils;
+use FileSystem;
+use MasonHelp;
+
+proc main() {
```

Comment:
While this works, I'd suggest switching these calls to direct spawn calls, so that you can check the exit code rather than depend on the help output. That way, we don't need to update this `.good` file every time we tweak the help output.

```chpl
use Spawn;

proc checkCommand(cmd: string) {
  var splitCmd = 'mason new -h'.split();
  var p = spawn(splitCmd);
  p.wait();
  if p.exit_status != 0 {
    writeln("Failed to run cmd: '%s'".format(cmd);
  }

  checkCommand('mason new -h');
  checkCommand('mason add -h');
  // ...
}

Then we can just have an empty `.good` file.